src/main/java/seedu/address/logic/commands/scorecommand/SubtractScoreCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands.scorecommand;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_TEAM_DISPLAYED_INDEX;
+
+import seedu.address.commons.exceptions.AlfredException;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.entity.Id;
+import seedu.address.model.entity.PrefixType;
+import seedu.address.model.entity.Score;
+import seedu.address.model.entity.Team;
+
+/**
+ * Subtracts a score from a team's current score.
+ */
+public class SubtractScoreCommand extends ScoreCommand {
+
+    public static final String MESSAGE_SCORE_TEAM_SUCCESS = "Subtracted %1$s points from %2$s's score";
+    public static final String COMMAND_WORD = "score sub";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": subtracts the specified score from the specified team's current score. "
+            + "If the team's new score goes below " + Score.MIN_SCORE + " it will be set to " + Score.MIN_SCORE + "\n"
+            + "Format: " + COMMAND_WORD + " teamID score \n"
+            + "For example: " + COMMAND_WORD + " T-5 15";
+
+    public SubtractScoreCommand(Id teamId, Score score) {
```

Comment:
hey what if score is given a negative value here. Would this then become addition

Reply:
the score can't be negative, so this wont happen
in fact that is why there is a subtract and add operation